conmon/conmon.c

```diff
@@ -869,10 +869,10 @@ static void setup_terminal_control_fifo()
 	int dummyfd = open(ctl_fifo_path, O_WRONLY | O_CLOEXEC);
```

Comment:
why not just set this as _cleanup_close_ ? Does it need to be in the scope of main to work?

Reply:
I believe so.
I will let Giuseppe judge, I think this has to be open for the entire run of conmon and only closed when it exits, but coverity sees this as a leak.

Reply:
yes, it needs to be in the scope of main, otherwise it will be closed as soon as we leave `setup_terminal_control_fifo`